src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadInstaller.cs

```diff
@@ -9,15 +9,15 @@ internal interface IWorkloadInstaller
     {
         InstallationUnit GetInstallationUnit();
 
-        void InstallWorkloadManifest(string manifestId, string manifestVersion, string sdkFeatureBand);
+        void InstallWorkloadManifest(ManifestId manifestId, ManifestVersion manifestVersion, SdkFeatureBand sdkFeatureBand);
```

Comment:
@wli3 Does this mean the manifest version is now a proper version? Last time I checked it was defined as a long.